engine/src/main/java/org/destinationsol/game/item/ItemManager.java

```diff
@@ -231,22 +233,40 @@ public void addAllGuns(ItemContainer ic) {
         }
     }
 
-    public List<MoneyItem> moneyToItems(float amt) {
-        ArrayList<MoneyItem> res = new ArrayList<>();
-        while (amt > MoneyItem.AMT) {
-            MoneyItem example;
-            if (amt > MoneyItem.BIG_AMT) {
-                example = moneyItem(MoneyItem.BIG_AMT);
-                amt -= MoneyItem.BIG_AMT;
-            } else if (amt > MoneyItem.MED_AMT) {
-                example = moneyItem(MoneyItem.MED_AMT);
-                amt -= MoneyItem.MED_AMT;
+    /**
+     * Turn money into item drops
+     * @param amount the amount of money to turn into drops
+     */
+    public List<MoneyItem> moneyToItems(float amount) {
+        return moneyToItems(amount, -1);
```

Comment:
I'm unsure about the -1. IMO, either set a sane default limit, or mark this method as  as `@Deprecated`, or even straightforward remove it

Reply:
I didn't want to bake-in a default value, because there might be enemies in the future who might need to drop more, even though this method is just used for the player right now. I also thought about checking out the physics, maybe items should not collide with each other (or maybe if there are too many items on screen), and then the limit could be removed later.

Reply:
If you need to drop more items, use the full function instead of this override IMO. The default behaviour should be sane when possible, esp. with the new code in modules stuff.

Reply:
Okay, set default behavior to 60 money items dropped, that should be okay for all needs.